src/sdltiles.cpp

```diff
@@ -969,12 +970,14 @@ void cata_cursesport::curses_drawwindow( const catacurses::window &w )
         update = true;
     } else if (g && w == g->w_pixel_minimap && g->pixel_minimap_option) {
         // Make sure the entire minimap window is black before drawing.
-        clear_window_area( w );
+        //        clear_window_area( w );
```

Comment:
Remove commented out code, may also need to require accompanying comment above it.